modules/sandboxed-containers-deleting-operator-web-console.adoc

```diff
@@ -0,0 +1,20 @@
+//Module included in the following assemblies:
+//
+// *uninstalling-sandboxed-containers.adoc
+
+[id="sandboxed-containers-deleting-operator-web-console_{context}"]
+= Deleting {sandboxed-containers-operator}
+
+You can delete the {sandboxed-containers-operator} by deleting the catalog subscription and removing namespace access to the Operator.
```

Comment:
I found "removing namespace access" to read awkwardly, but maybe there's precedent for those words in that order.  If not, what about "revoking namespace access" since permissions tend to be granted, denied, and revoked?